src/modules/item/utils.ts

```diff
@@ -461,6 +463,51 @@ export function areSynced(item: Item, entity: Entity) {
   return true
 }
 
+export function isEmoteSynced(item: Item, entity: Entity, isEmotesFeatureFlagOn: boolean) {
+  if (item.type !== ItemType.EMOTE) {
+    throw new Error('Item must be EMOTE')
+  }
+
+  // check if metadata has the new schema from ADR 74
+  const isADR74 = 'emoteDataADR74' in entity.metadata
+  if (!isADR74 && isEmotesFeatureFlagOn) {
+    return false
+  }
+
+  // check if metadata is synced
+  const catalystItem = entity.metadata! as Emote
+
+  const hasMetadataChanged =
+    item.name !== catalystItem.name ||
+    item.description !== catalystItem.description ||
+    (item.data.category as string) !== catalystItem.emoteDataADR74.category ||
```

Comment:
just a heads up, this is throwing for me, not being able to access `category` of `undefined`. This is because I have some legacy emotes deployed, and the `isEmotesFeatureFlagOn` is set to `false`. I think if that flag is false, then we should NOT look for the `emoteDataADR74`, otherwise owners of legacy emotes won't be able to use the builder until the flag is on. Instead we can look for the `emoteDataV0` prop while the flag is off, or just return false, which is not great, but is better than getting an error.

cc @juanmahidalgo 

<img width="944" alt="Screen Shot 2022-08-10 at 16 04 55" src="https://user-images.githubusercontent.com/2781777/183996584-59d75fe5-ea7f-4ca7-a6e5-b160922bbb07.png">